Cython/Compiler/Symtab.py

```diff
@@ -1351,6 +1352,17 @@ def declare_builtin(self, name, pos):
         entry.qualified_name = self.builtin_scope().qualify_name(name)
         return entry
 
+    def _is_package_scope_or_module(self):
+        # Returns True for all ModuleScopes representing package or scopes representing
+        # modules. Otherwise returns False.
+        # Note: For package pkg_a Cython creates two modules scopes: pkg_a and pkg_a.__init__.
```

Comment:
I wonder if it's even correct that we create two separate module scopes. And a good thing. It seems to me that the answer might be no.

Reply:
Yeas I agree. I fixed in in next version of this PR - see below.